### ISSUE TITLE
Composition date range larger than composer's life span.

### ISSUE THREAD

**Issue by alexandermorgan** [ELVIS-Project/elvis-database#152]
I'm not sure if this is the right place for this issue. A mass by Du Fay has a date range from 1307-1474. But it was most likely meant to be 1407 (i.e. ten years after his birth) to 1474 (his death). Perhaps we could not allow a piece contributed to a composer to have a composition date range that lies outside of the composer's lifetime. Also a suggested range of 1460-1465 was recently proposed by Jesse Rodin ("The L'homme arme tradition" in Fifteenth Century Music, 2015). Here's the database link to the piece:
http://database.elvisproject.ca/piece/467/

**Comment by AFFogarty** [comment]
Isn't it possible that a piece could be published posthumously?

**Comment by alexandermorgan** [comment]
Yes, good point but our dates are composition dates I believe. If we want to go by publication date that would be a valid approach too but we'd have to specify that.
